### PR TITLE
Feat: Added counter cart

### PR DESCRIPTION
**Pull request by anjali1102**

### CODE REVIEW

**Comment by sainath7878** [comment]
Screenshot 2022-06-02 at 11 32 09 AM

Check the footer styling.

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 import { Link } from "react-router-dom";
+import { useWishlist } from "../../context/wishlist/WishlistContext";
+import { useCart } from "../../context/cart/CartContext";
+import "./Navbar.css";
+import { FaHeart, FaShoppingCart, FaUserCircle } from "react-icons/fa";
 
 const Navbar = () => {
+  const { wishlist } = useWishlist();
+  const { cart } = useCart();
+  // console.log("WishlistState ", wishlistState);
```
Comment (sainath7878) [inline src/components/Navbar/Navbar.js]: remove commented code

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+  console.log("cart ", cart);
+
+  // console.log("wishlist", wishlist);
```
Comment (sainath7878) [inline src/components/Navbar/Navbar.js]: here too

**Review by dev-enforced — APPROVED** [review]

File: src/pages/ProductPage/ProductPage.js [hunk 3]
```diff
@@ -15,14 +15,14 @@ import { useCart } from "../../context/cart/CartContext";
 
 import toast, { Toaster } from "react-hot-toast";
 
-const notify = () => toast.success("Added to Card!");
+const notify = () => toast.success("Added to Card 👜 !!");
+const notify2 = () => toast.success("Added to Wishlist 💗 !! ");
```
Comment (sainath7878) [inline src/pages/ProductPage/ProductPage.js]: suggestion: naming could be better

**Review by sainath7878 — APPROVED** [review]
Check the suggested changes. You've console.log written at many places in the code. Make sure to remove them. Rest looks good. Nice work on the UI.

File: src/pages/LoginPage/LoginPage.css [hunk 4]
```diff
@@ -1,12 +1,8 @@
 .center {
-  width: 400px;
   flex: 1;
   margin-block: 2rem;
   background: white;
   border-radius: 10px;
-  /* box-shadow: -8px 17px 27px -5px rgba(0, 0, 0, 0.78); */
-  /* -webkit-box-shadow: -8px 17px 27px -5px rgba(0, 0, 0, 0.78); */
-  /* -moz-box-shadow: -8px 17px 27px -5px rgba(0, 0, 0, 0.78); */
```
Comment (jeetsdev) [inline src/pages/LoginPage/LoginPage.css:9]: Remove commented codes

File: src/pages/ProductPage/ProductPage.js [hunk 5]
```diff
@@ -15,14 +15,14 @@ import { useCart } from "../../context/cart/CartContext";
...
 
 const ProductPage = () => {
   const [state, dispatch] = useReducer(filterReducer, defaultFilterState);
   const { wishlist, dispatchWishlist } = useWishlist();
   const { cart, dispatchCart } = useCart();
   const [products, setProducts] = useState([]);
-  // console.log({ wishlist });
```
Comment (jeetsdev) [inline src/pages/ProductPage/ProductPage.js:25]: Remove commented codes.

**Review by jeetsdev — APPROVED** [review]
Everything looks good.
Code and modular and clean.
Good folder structure.

**Pull request merged**